MAUI/TreeView/selection.md

```diff
@@ -0,0 +1,167 @@
+
+When the selection mode is `Single` or `SingleDeselect`, programmatically select an item by setting the underlying object to the `SelectedItem` property.
+
+{% tabs %}
+{% highlight c# %}
+
+treeView.SelectedItem = viewModel.CountriesInfo[2];
+
+{% endhighlight %}
+{% endtabs %}
+
+When the selection mode is `Multiple`, programmatically select more than one item by adding the underlying object to the `SelectedItems` property.
+
+{% tabs %}
+{% highlight c# %}
+
+treeView.SelectedItems.Add(viewModel.CountriesInfo[2]);
+treeView.SelectedItems.Add(viewModel.CountriesInfo[3]);
+
+{% endhighlight %}
+{% endtabs %}
+
+W> If an item is selected programmatically when `SelectionMode` is `None` and if multiple items are programmatically selected when `SelectionMode` is `Single` or `SingleDeselect`, then exception will be thrown internally.
+
+## Selected items 
+
+### Gets selected Items
+The TreeView gets all the selected items through the `SelectedItems` property and gets the single item by using the `SelectedItem` property.
+
+### Clear selected items
+The selected items can be cleared by calling the `SelectedItems.Clear()` method.
+
+{% tabs %}
+{% highlight c# %}
+treeView.SelectedItems.Clear();
+{% endhighlight %}
+{% endtabs %}
+ 
+### CurrentItem vs SelectedItem
+
+The TreeView gets the selected item by using the `SelectedItem` properties. Both `SelectedItem` and `CurrentItem` returns the same data object when selecting single item. When selecting more than one item, the `SelectedItem` property returns the first selected item, and the `CurrentItem` property returns the last selected item.
+
+## Select an entire row
+
+By default, the selection starts from the indent level only. You can select the full row by enabling the `FullRowSelect` property. By setting the `FullRowSelect` to `true` the selection spans the width of tree view control.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfTreeView x:Name="TreeView" FullRowSelect="True" />
+{% endhighlight %}
+{% highlight c# %}
+TreeView.FullRowSelect = true;
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI TreeView with FullRowSelect](Images/selection/FullRowSelect.png)
+
+## Selected item style
+
+### Selection background
+
+The TreeView allows changing the selection background color for the selected items by using the `SelectionBackgroundColor` property. You can also change the selection background color at runtime.
+
+### Selection foreground
+
+The TreeView allows changing the selection foreground color for the selected items by using the `SelectionForegroundColor` property. You can also change the selection foreground color at runtime.
+
+N> `SelectionForegroundColor` is applicable only for unbound mode.
+
+## Events
+
+### SelectionChanging Event
+
+The `SelectionChanging` event is raised while selecting an item at the execution time. The `ItemSelectionChangingEventArgs` has the following members which provides the information for `SelectionChanging` event:

```

Comment:
The `SelectionChanging` event is raised while selecting an item at the execution time. The `ItemSelectionChangingEventArgs` has the following members which provide the information for `SelectionChanging` event: